modules/AOR_Reports/AOR_Report.php

```diff
@@ -955,7 +955,7 @@ public function getTotalHTML($fields, $totals)
         $currency->retrieve($GLOBALS['current_user']->getPreference('currency'));
 
         $showTotal = false;
-        $html = '<table>';
+        $html = '<table width="100%" class="list view table-responsive aor_reports">';
```

Comment:
Not necessarily required to merge but it would be better to separate this out into a template and use smarty.